webapi/src/it/scala/org/knora/webapi/e2e/v2/OntologyV2R2RSpec.scala

```diff
@@ -401,6 +401,10 @@ class OntologyV2R2RSpec extends R2RSpec {
 
           Get(httpGetTest.urlPath).addHeader(Accept(mediaType)) ~> ontologiesPath ~> check {
             val responseStr: String = responseAs[String]
+            val isOkResponse        = response.status == StatusCodes.OK
+            if (!isOkResponse) {
+              println(httpGetTest)
+            }
```

Comment:
yes in this case it is very helpful. That is because this particular test is a kind of "parameterized" test, i.e. it is executing the same test for a lot of requests. It failed during development and I did not now which request was the perpetrator. This println will give the information in the case of a failure.